src/net/peer/connect/bootstrap/mod.rs

```diff
@@ -16,8 +16,8 @@
 // relating to use of the SAFE Network Software.
 
 mod cache;
-mod bootstrap;
+mod bootstrap_fn;
```

Comment:
Naming module after function (`_fn`) sounds very specific, doesn't it?
What if we simply moved `bootstrap()` function out of `bootstrap` module to `mod.rs`?

Reply:
Ugh, we could. I just prefer to keep code in leaf modules rather than putting it in `mod.rs`. I don't like the naming here either though.